includes/wc-core-functions.php

```diff
@@ -627,7 +627,7 @@ function get_woocommerce_currency_symbol( $currency = '' ) {
 			'USD' => '&#36;',
 			'UYU' => '&#36;',
 			'UZS' => 'UZS',
-			'VEF' => 'Bs F',
+			'VEF' => 'Bs S',
```

Comment:
@zJesusJavier can you place restore it and include:

```php
'VES' => 'Bs.S',
```

Also need to include the currency in `get_woocommerce_currencies()`, need to include into the list the follow code:

```php
'VES' => __( 'Bolívar soberano', 'woocommerce' ),
```

Reply:
Working fine with those changes:

![annotation 2019-01-15 113627](https://user-images.githubusercontent.com/39272785/51191075-0649e300-18ba-11e9-8a99-8dcb564e4964.jpg)
![annotation 2019-01-15 113736](https://user-images.githubusercontent.com/39272785/51191076-0649e300-18ba-11e9-9cb7-682f27eaca30.jpg)

I added an space at the end of the new currency, since looks better to have the amount separated from the currency symbol.

Reply:
Please do not include empty spaces, there is settings for it.